gems/spree_auth_devise/GHSA-6mqr-q86q-6gwr.yml

```diff
@@ -0,0 +1,35 @@
+---
+gem: spree_auth_devise
+ghsa: 6mqr-q86q-6gwr
+url: https://github.com/solidusio/solidus_auth_devise/security/advisories/GHSA-xm34-v85h-9pg2
+title: Authentication Bypass by CSRF Weakness
+date: 2021-11-18
+description: "### Impact\n\nCSRF vulnerability that allows user account takeover.\n\nAll
```

Comment:
Please use `description: |` so that the text isn't an escaped double-quoted String, which makes it easier to read and edit.